### DISCUSSION TITLE
Bulk editing in Inbox

### DISCUSSION THREAD

**Discussion by lagset** [astubenbord/paperless-mobile#222] [Feature Requests]
Bulk editing works great in the app! I would like to use these tools in the inbox as well, especially when adding a bunch of documents from the same source.

Currently I need to change to the documents view and sort by *added* to be able to tag my newly added documents.

**Comment by astubenbord** [comment]
Hi, I'll have a look and see how much effort this would be to also support bulk editing in the inbox.
